doc/src/devdocs/subarrays.md

```diff
@@ -36,14 +36,37 @@ cartesian, rather than linear, indexing.
 
 Consider making 2d slices of a 3d array:
 
-```julia
-S1 = view(A, :, 5, 2:6)
-S2 = view(A, 5, :, 2:6)
+```@meta
+DocTestSetup = :(srand(1234))
+```
+```jldoctest subarray
```

Comment:
In line 179, should `jldoctest` -> `jldoctest subarray` like here? (I couldn't figure out what this really does from the Documenter.jl docs)